depends/builders/freebsd.mk

```diff
@@ -1,5 +1,5 @@
 build_freebsd_CC=clang
 build_freebsd_CXX=clang++
 
-build_freebsd_SHA256SUM = shasum -a 256
+build_freebsd_SHA256SUM = sha256sum
```

Comment:
Doesn't this apply to [NetBSD](https://github.com/bitcoin/bitcoin/blob/7b06ffce9c50110b475c722918c55a14402346a5/depends/builders/netbsd.mk#L1) as well?

Reply:
No, it doesn't, unfortunately.

NetBSD's [`sha256`](https://man.netbsd.org/sha256.1) is not quite compatible with our depends build syytem.